plugins/generator-1.19.4/forge-1.19.4/utils/triggers.java.ftl

```diff
@@ -1,6 +1,25 @@
 <#include "procedures.java.ftl">
 
 <#-- Item-related triggers -->
+<#macro addSpecialInformation procedure="" isBlock=false>
+	<#if procedure?has_content || hasProcedure(procedure)>
+		@Override public void appendHoverText(ItemStack itemstack, <#if isBlock>BlockGetter<#else>Level</#if> world, List<Component> list, TooltipFlag flag) {
+		super.appendHoverText(itemstack, world, list, flag);
+		<#if hasProcedure(procedure)>
+			Entity entity = itemstack.getEntityRepresentation();
+			double x = entity != null ? entity.getX() : 0.0;
+			double y = entity != null ? entity.getY() : 0.0;
+			double z = entity != null ? entity.getZ() : 0.0;
+			list.add(Component.literal(<@procedureOBJToStringCode procedure/>));
+		<#else>
+			<#list procedure.getFixedValue() as entry>
+				list.add(Component.literal("${JavaConventions.escapeStringForJava(entry)}"));

```

Comment:
Got me thinking, we actually can add a `localizationkeys` entry that would take strings from `data.specialInformation.getFixedValue()` and generate translations for them... Separate PR, though? (If yes, I think I can do it after this one)

Reply:
Separate PR indeed, this one is more of procedures in translation tips.

But would be neat addition